wazuh/wazuh-kibana-app
# Modify Overview application to display the new menu

### Description

Change the Overview page to display new menu apps. Additional change icons in the menu.
 
### Issues Resolved

https://github.com/wazuh/wazuh-kibana-app/issues/5835

### Evidence

![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/3c9fc172-12d5-4c8c-8ff4-43d1b2d294a1)
![image](https://github.com/wazuh/wazuh-kibana-app/assets/124377319/64fc4673-3aa1-449c-be4e-c40b5e66cddc)


### Test
- Run environment.
- Check if the overview page is functional.

### Check List
